packages/codefixes/src/codefixes.ts

```diff
@@ -25,7 +26,11 @@ export const codefixes = new CodeFixesProvider([
 ]);
 
 export const glintCodeFixes = new CodeFixesProvider([
-  new BaseCodeFixCollection([new AddMissingArgToComponentSignature()]),
+  new GlintCodeFixCollection(),
+  new BaseCodeFixCollection([
+    // Need to be run after standard "typedef to type" fix applied
+    new AddMissingArgToComponentSignature(),
```

Comment:
Why would this not be in GlintCodeFixCollection? I understand it needs to be last.

Reply:
Glint collection provides codefixes provided by Glin service (typescript ones + glint ones). Base collection is for our own.